goaway.go

```diff
@@ -77,11 +77,18 @@ func (g *ProfanityDetector) WithCustomDictionary(profanities, falsePositives, fa
 // IsProfane takes in a string (word or sentence) and look for profanities.
 // Returns a boolean
 func (g *ProfanityDetector) IsProfane(s string) bool {
+	_, profane := g.IsProfaneWord(s)
+	return profane
+}
+
+// IsProfane takes in a string (word or sentence) and look for profanities.
+// Returns the first offending word and a boolean
+func (g *ProfanityDetector) IsProfaneWord(s string) (string, bool) {
```

Comment:
Would is be possible to rename this to `ExtractProfanity` or something similar? 

Could also just return a string; if the len() of the string returned is > 0, then it pretty much means `true` already.

Reply:
* I'm fine with the rename, That was just a really quick name i came up with so i could get it implemented.
* Its a little easier to use, in my opinion, if you aren't expected to check string length for profanity, and you could ignore the word. Though if you feel strongly about it only returning the string, i can make that implementation change, since at the end of the day its your project.

Reply:
I'm fine with keeping the bool then, so just the rename then :)

Reply:
already done